LICENSE.md

```diff
@@ -400,3 +400,1477 @@ License: [The Apache Software License, Version 2.0](http://www.apache.org/licens
 
 ===========================================================================
 
+### [MapboxCoreMaps](https://www.mapbox.com/) by Mapbox
```

Comment:
the format seems a bit different from the above, if the license is just Apache or BSD or Mapbox ToS, shall we just add the url as above, instead of pasting the whole license?